src/app/plugins/twitter.plugin.ts

```diff
@@ -0,0 +1,103 @@
+import { MessageEmbed } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage } from '../../common/types';
+import { TwitterTimelineResponse, TwitterService } from '../../services/twitter.service'
+
+const accounts: Record<string, string> = {
+    ucf: '18999501',
+    knights: '21306514',
+    football: '30282826',
+    knighthacks: '3122136832',
+    cecs: '2292877801',
+}
+
+const twitterIconURL = 'https://images-ext-1.discordapp.net/external/bXJWV2Y_F3XSra_kEqIYXAAsI3m1meckfLhYuWzxIfI/https/abs.twimg.com/icons/apple-touch-icon-192x192.png';
+
+export class TwitterPlugin extends Plugin {
+
+    // The twitter API returns a min of 5 tweets, but thats a bit much for our bot, so we'll do 3 instead.
+    maxSize = 3;
+
+    public get name(): string {
+        return 'twitter'
+    }
+    public get description(): string {
+        return 'Gets the latest twitter timelines from UCF accounts';
+    }
+    public get usage(): string {
+        throw 'twitter <UCF account>';
+    }
+
+    public get permission(): ChannelType {
+        return ChannelType.Public
+    }
+
+    private twitter = new TwitterService();
+
+    public constructor(public container: IContainer) {
+        super();
+    }
+
+    public async execute(message: IMessage, args: string[]) {
+        const [param] = args;
+
+        let accountId;
+
+        // Default to UCF account if no args provided.
+        if (args.length === 0) {
+            accountId = accounts.ucf;
+        } else {
+            accountId = accounts[param.toLowerCase()];
+        }
+
+        // Show possible options if invalid account was specified.
+        if (!accountId) {
+            let options = '\n';
+            Object.keys(accounts).forEach(key => options += `🔸 ${key}\n`);
+            message.react('❌')
+            message.reply(`Invalid UCF Twitter Account \'${param}\', possible options are:${options}`);
+            return;
+        }
+
+        message.react('👍');
+        message.reply('Sure thing! Getting latest tweets!');
+
+        // Fetch respective tweets.
+        const response = await this.twitter.getLatestTweets(accountId, this.maxSize);
+        const embeds = await this._createEmbedList(response, accountId);
+
+        // Send out all of the fetched tweets.
+        embeds.forEach(embed => message.reply(embed));
```

Comment:
How many embeds can this be?
We may want to combine them into one embed so that we dont spam / get rate limited

Reply:
This has a max of 3 embeds. Combining them all together is less than ideal since we'll lose a lot of rich content. However I've now adjusted the code to use a webhook in order to send all three in a single payload as opposed to 3 different payloads, so it shouldn't affect rate limits.